Description: remove tidyVersion as it is a special call
 to Andy Lester's modified version of libtidy. Also remove
 the corresponding call from Perl, the documentation, and
 the tests.
Author: gregor herrmann <gregoa@debian.org>
Reviewed-by: Jonathan Yu <jawnsy@cpan.org>
Origin: vendor
Forwarded: not-needed

--- a/Tidy.xs
+++ b/Tidy.xs
@@ -189,14 +189,3 @@
         if ( rc < 0 ) {
             XSRETURN_UNDEF;
         }
-
-
-SV*
-_tidy_version()
-    PREINIT:
-        const char* version;
-    CODE:
-        version = tidyVersion();
-        RETVAL = newSVpv(version,0); /* will be automatically "mortalized" */
-    OUTPUT:
-        RETVAL
--- a/lib/HTML/Tidy.pm
+++ b/lib/HTML/Tidy.pm
@@ -339,18 +339,6 @@
     return 1;
 }
 
-=head2 libtidy_version()
-
-Returns the version of the underling tidy library.
-
-=cut
-
-sub libtidy_version {
-    my $version_str = _tidy_version();
-
-    return $version_str;
-}
-
 require XSLoader;
 XSLoader::load('HTML::Tidy', $VERSION);
 
--- a/t/version.t
+++ b/t/version.t
@@ -5,6 +5,8 @@
 use Test::More;
 
 BEGIN {
+    plan skip_all => 'libtidy_version has been removed in Debian';
+
     plan tests => 3;
     use_ok( 'HTML::Tidy' );
 }
